WcaOnRails/app/models/competition.rb

```diff
@@ -195,6 +195,8 @@ class Competition < ApplicationRecord
            allow_nil: true,
            with_model_currency: :currency_code
   validates :early_puzzle_submission_reason, presence: true, if: :early_puzzle_submission?
+  # cannot validate `qualification_results IN [true, false]` because we historically have competitions
```

Comment:
Is it worth having a nullable boolean, versus just defaulting to false for old competitions?

Reply:
Neat thought that I also had while working on this fix. But changing the column default of an already existing column feels out of the scope of this PR, especially given how urgent it is.